test/api/fonts/README

```diff
@@ -3,3 +3,5 @@ cmap-format12-only files created by ttx & remove all other cmap entries
 Inconsolata-Regular.abc.widerc.ttf has the hmtx width of "c" set to 600; everything else is 500. Subsetting out c should reduce numberOfHMetrics to 1.
 
 chromacheck-* fonts are from https://github.com/RoelN/ChromaCheck/tree/master/fonts and licensed under MIT by Roel Nieskens and Google.
+
+RanaKufi-Regular.subset.otf is from https://github.com/alif-type/rana-kufi/ and licensed under AGPLv3
```

Comment:
Is it okay to use specially license-wise

Reply:
Feel free to use that file under any suitable license. I didn't check this particular test, but I’m using the latest code and didn't notice any issue.

Reply:
Thank you :)